commons-executors/src/main/java/com/palantir/common/concurrent/PTExecutors.java

```diff
@@ -490,41 +492,24 @@ public static Runnable wrap(final Runnable runnable) {
         return new ForwardingRunnableFuture<>(wrappedRunnable, rf);
     }
 
-    private static Runnable wrapRunnable(Runnable runnable) {
+    /**
+     * Wraps the given {@code Callable} so that {@link ExecutorInheritableThreadLocal} variables are
+     * propagated through.
+     */
```

Comment:
👍

Reply:
Huh, this was just a method reordering, but it seems the methods are rather similar 😂

Reply:
Yep. Realised that after the fact - but still, good that what this method does is explained